book/07-git-tools/sections/rerere.asc

```diff
@@ -1,31 +1,31 @@
-One of the examples that is mentioned in the documentation is when you want to make sure a long-lived topic branch will ultimately merge cleanly, but you don't want to have a bunch of intermediate merge commits cluttering up your commit history.
-With `rerere` enabled, you can attempt the occasional merge, resolve the conflicts, then back out of the merge.
-If you do this continuously, then the final merge should be easy because `rerere` can just do everything for you automatically.
+Mayroong mga bilang ng mga sitwasyon na kung saan ang pag-andar nito ay maaaring talagang magaling.
+Isa sa mga halimbawa na nabanggit sa dokumentasyon ay kapag gusto mo na tiyakin ang isang matagal na paksa na branch ay ganap na pagsasama nang malinis, ngunit ayaw mong makakaroon ng isang grupo ng intermediate merge na mga commit na nag-clutter pataas sa iyong kasaysayan ng commit.
+Sa `rerere` na pinagana, maaari kang magtangka sa minsang na-merge, malutas ang mga pagkasalungat, pagkatapos ay i-atras ang pag-merge.
+Kung gagawin mo ito nang paulit-ulit, pagkatapos ay ang huling merge ay dapat na madali dahil sa `rerere` ay maaari lamang gawin ang lahat para sa iyo ng awtomatiko.
 
-This same tactic can be used if you want to keep a branch rebased so you don't have to deal with the same rebasing conflicts each time you do it.
-Or if you want to take a branch that you merged and fixed a bunch of conflicts and then decide to rebase it instead -- you likely won't have to do all the same conflicts again.
+Ang parehong taktika ay maaaring gamitin kung gusto mong itago ang isang branch na naka-rebase rebased kaya hindi mo kailangang harapin ang parehong pag-rebase sa mga sumasalungat sa bawat oras na ginawa mo ito.
+O kung gusto mong kumuha ng isang branch na  if you want to take a branch na nai-merge mo at nag-ayos ng maraming mga pagkakasalugat at pagkatapos ay magpasya na i-rebase ito sa halip -- malamang na hindi mo kailanganin gawin muli ang parehong mga pagkakasalungat muli.
 
-Another application of `rerere` is where you merge a bunch of evolving topic branches together into a testable head occasionally, as the Git project itself often does.
-If the tests fail, you can rewind the merges and re-do them without the topic branch that made the tests fail without having to re-resolve the conflicts again.
+Isa pang application sa `rerere` na kung saan ikaw ay mag merge ng maraming nagbabago na paksang mga branch magkasama sa isang pagsubok sa ulo pangminsan-minsan, bilang ang Git na proyekto sa kanyang sarili ay madalas na ginagawa.
+Kung ang pagsubok ay nabigo, maaari mong i-rewind ang mga merge at gawin muli sila ng walang paksa sa branch na ginawa mong pagsubok na walang nareresulba na mga pagsasalungat muli.
 
-To enable `rerere` functionality, you simply have to run this config setting:
+Para mapagana ang `rerere` na pag-andar, kailangan mo lang na ipagtakbo ang config na setting na ito:
```

Comment:
`ipagtakbo` to `ipatakbo` or `patakbuhin`

Reply:
done change